platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md

```diff
@@ -316,6 +316,22 @@ You bring:
   - **TestRail** [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **OR**
   - **Markdown** [Github accessibility test cases template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md)
 - List of known issues (with links to tickets when applicable)
+- On the request template, certify that the following accessibility checks are complete:
+    - Read the accessibility staging review documentation (required)
+    - Read the accessibility post-launch processes (recommended)
+    - Conduct axe browser scan, document or address issues found (required)
```

Comment:
I'd like to unpack this point in a bit more detail. At a minimum, devs must fix all axe `violations` because that's what's defined in Section 508 (WCAG 2 Level A and Level AA violations). It's also what we're obligated to do for interim launch. Devs may or may not be able to fix the best practices. Ideally they fix the ones they can (heading nesting for instance) but those are nice to haves, not requirements.

* https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md#automated-testing-required

Reply:
Add a bullet point, for example:

- Fix axe violations before staging review

Although, how will teams identify an axe violation from a best practice? The axe browser scan doesn't make it clear, in my experience, relative to the severity rubric. Some are listed as Critical, but only rate a defect level 3 for us. 

I'd like to see them log the issues they find, and then we are able to let them know at the staging review which **must** be fixed before launch.

Reply:
Violations have to be fixed, or the branch can't be merged into `master` if it's static content, and will break e2e tests assuming good axe check coverage. The axe plugin for browsers has a dropdown to filter findings by All, Violations, Needs Review, Best Practices. We could improve language on this to help teams identify what are violations.

I can also explicitly add axe check violations to our `1` launch blocking issues. This identifies an assumption on my part.

Reply:
Staging review, to my eyes, seems more front-end and design centered. 

For the staging review, we won't have access to the codebase violations, will we? If we could have axe check coverage QA as part of the staging review, that would be first-rate. I thought that was part of the engineering checks. In a v2 of the VSP CC staging review, perhaps engineering could document what needs to be covered.